RELEASENOTES.md

```diff
@@ -1,5 +1,120 @@
+- Unless the application has been preconfigured, you will need to configure the application environments. The application will prompt the user if the configuration file is missing or is invalid.
+
+## How to Configure
+After the client is installed, you may get a prompt that config.json file is missing.
+* If ApplicationBrowser-<version>.msi was used to install Application Browser, place the config.json file under C:\Program Files (x86)\Application Browser
+* If ApplicationBrowser-<version>-setup.exe was used to install Application Browser, place the config.json file under C:\Users\<username>\AppData\Local\ApplicationBrowser\app-<version>
+Create this file and populate it with environment URLs. Here is a sample configuration file.
+```
+{
+  "flash":
+  {
+    "path":"C:\\Windows\\System32\\Macromed\\Flash\\pepflashplayer64_28_0_0_137.dll"
+  },
+
+ "environments":
+  [
+    {
+      "id": "test",
+      "url": "https://www.google.it",
+      "label": "Test"
+    }, {
+      "id": "sandbox",
+      "url": "https://us.yahoo.com/",
+      "label": "Sandbox"
+    }, {
+      "id": "production",
+      "url": "https://www.facebook.com",
+      "label": "Production"
+    }
+  ],
+
+  "whenUrl": [
+    {
+      "endsWith": "PPSS/PPSS",
+      "thenAppend": "?control=loginPage"
+    }
+  ]
+}
+```
+
+* flash > path (optional) - The path to the installed pepper flashplayer.dll file. The application will attempt to use this path to the Pepper Flash Player. If not found, the application will attempt to find an installed version.
+* environments (required) - An array of environments that will be available on the initial page. Each environment has the following properties:
+* id (required) – A unique id for the environment.
+* url (required) – A fully qualified path to a website. When configuring PPSS URLs, be sure to configure the fully qualified path to the login page. A typical PPSS configuration will look like https://server-alpha:8123/PPSS/PPSS?control=loginPage
+* label (required) - The name of the link that will be shown in the UI. This label is also used in the title upon opening an environment
+* develop (optional) - A Boolean value used to turn on development mode. This is not recommended for production installations.
+* locale (optional) – A locale value that will override the system locale. This is not recommended for production installations.
+* whenUrl (optional) – A set of conditions used to update a URL when opened from the main window.
+	* endsWith – A string to look for in the URL when loaded
+	* thenAppend – A string to append to the end of a URL if the endsWidth string is found.
+After the application is successfully opened, it creates the launchpage-config.json file in the user’s folder. This file has the hide property for each configured environment that indicates if this environment is hidden or not. The user will be able to override this value within the running application. Here is a sample of this file:
```

Comment:
This should be new paragraph, right now it is appended to thenAppend.
![image](https://user-images.githubusercontent.com/73573586/98389808-2cee8600-201a-11eb-9afe-643086d6fd7e.png)